explorer/interpreter/value.cpp

```diff
@@ -291,6 +291,9 @@ void Value::Print(llvm::raw_ostream& out) const {
     case Value::Kind::BoolValue:
       out << (cast<BoolValue>(*this).value() ? "true" : "false");
       break;
+    case Value::Kind::DestructorValue:
+      out << "destructor [ me: Self ] ";
```

Comment:
Can't destructors also be declared with `[addr me: Self*]`?

Reply:
Done